tests/bootstrap.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
-declare(strict_types=1);
```

Comment:
if we want to manage these files via dev-kit, we either need to remove this and support our lowest PHP version (organization wide) OR need to add it as a template to dev-kit, parse it and do some if-clauses in the template, depending on the "default project php version"